openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -29,6 +29,7 @@
     'marketing_security_group': 'mass_mailing_security_group',
     # odoomrp/odoomrp-wip
     'product_variants_no_automatic_creation': 'product_variant_configurator',
+    'sale_documents_comments': 'sale_comment_propagation',
```

Comment:
It has been already done in other odoomrp-wip modules. See https://github.com/OCA/OpenUpgrade/blob/3243e327e903bd0b24db41f9cf922bc2cdb7fc20/openerp/addons/openupgrade_records/lib/apriori.py#L30